test/parallel/test-console-is-a-namespace.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+// https://heycam.github.io/webidl/#es-namespaces
+// https://console.spec.whatwg.org/#console-namespace
+// https://github.com/w3c/web-platform-tests/blob/master/console/console-is-a-namespace.any.js
+
+const common = require('../common');
+const assert = require('assert');
+const { test, assert_equals, assert_true, assert_false } = common.WPT;
+
+assert.doesNotThrow(() => {
```

Comment:
@watilde I removed the duplicate. Also there is a [comment](https://github.com/Wandalen/node/blob/console-consistent-with-standard/test/parallel/test-console-is-a-namespace.js#L15), explaining the test case above is not from WPT. The first test case is useful. I fails with original version of booststrap_node.js. Do you suggest to move the first test case into separate test suite?